.github/workflows/benchmarks/plot_results.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Acknowledging the approach for plotting from the quantum-benchmarks repository
```

Comment:
Actually, maybe if the code is taken exactly from that repo, we should include the licence here too (sorry!)

Reply:
No worries, thanks! Updated